subprojects/plugin-use/src/main/java/org/gradle/plugin/use/internal/DefaultPluginRequestApplicator.java

```diff
@@ -199,7 +204,9 @@ private void applyPlugin(PluginRequestInternal request, PluginId id, Runnable ap
         try {
             try {
                 applicator.run();
-                request.getAcceptanceHandler().run();
+                if (request.getOrigin() == AUTO_APPLIED) {
+                    autoAppliedPluginRegistryBroadcaster.autoApplied(request);
+                }
```

Comment:
❔ Why not call the listener for all methods, and do the check for auto-applied in the listener?

Reply:
Done in d0a9b7267ba79b9d3688a590893eb3bea7779ce4.